package.json

```diff
@@ -28,11 +52,38 @@
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-deprecation": "^1.3.2",
     "eslint-plugin-tsdoc": "^0.2.16",
+    "glob": "^8.0.3",
```

Comment:
Is this in use?

Reply:
Let's create an issue to clean up any unused dependencies later instead of tackling this every PR.
There are tools that detect this, but those can be added later too.